__init__.py

```diff
@@ -44,7 +44,8 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
-    os.path.join(HOME_DIR, 'scripts/'),
+    'scripts',
+    os.path.join(HOME_DIR, 'scripts/')
 ]
```

Comment:
I think that existed because earlier, users could create a script and keep it in the home directory. With the current setup, there is no need to search the home dir, either the scripts are in `.retriever/scripts`, or they exist inside the `scripts` fodler in the repo.